axelrod/strategies/__init__.py

```diff
@@ -99,18 +100,14 @@ def filtered_strategies(filterset, strategies=all_strategies):
     Applies the filters defined in the given filterset dict and returns those
     strategy classes which pass all of those filters from the given list of
     strategies.
-
```

Comment:
Why delete these lines?

Reply:
It probably got introduced, when I copied the raw file from git to get the dependencies back in the same order. 
Fixed it now. 👍